GMX-Bots/Unusual_PnL/src/network.ts

```diff
@@ -0,0 +1,41 @@
+export interface NetworkData {
+  vault: string;
+  unUsualLimit: number;
+  highPnlToSize: number;
```

Comment:
is better to use strings in this parameter, to ensure you get the number you want when using bignumberjs